setup.cfg

```diff
@@ -85,8 +85,6 @@ datasets =
     pyvista>=0.25.2,<0.39
     # radiant-mlhub 0.2.1+ required for api_key bugfix:
     # https://github.com/radiantearth/radiant-mlhub/pull/48
-    # radiant-mlhub 0.5+ changed download behavior:
-    # https://github.com/radiantearth/radiant-mlhub/pull/104
     radiant-mlhub>=0.2.1,<0.5
```

Comment:
```suggestion
    radiant-mlhub>=0.2.1,<0.6
```

Reply:
Also need to update environment.yml

Reply:
Is the above change to environment.yml sufficient?

Reply:
I think you understood the reverse of my intentions.

With this PR, the correct dependency versions are `>=0.2.1,<0.6`. This should be the case in all files, except `requirements/datasets.txt` where we still need to test the old version until newer shapely is supported. So you should still remove the upper bound in `environment.yml`

Reply:
Got it! Sorry for the confusion. How about now?